.github/workflows/lint.yml

```diff
@@ -96,4 +96,23 @@ jobs:
           sarif_file: eslint-results.sarif
           wait-for-processing: true
   # List future lints here only the linting command/uploads is needed
-  #  see eslint job above
\ No newline at end of file
+  #  see eslint job above
+
+  # Stylelint is a tool for identifying and reporting on patterns
+  # found in CSS code.
+  # More details at https://stylelint.io/
+  stylelint:
+    name: Stylelint ✨
+    runs-on: ubuntu-latest
+    needs: setup
+    strategy:
+      matrix:
+        node-version: [16, 18]  
+    steps:
+      - name: Load Cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ env.CACHE_PATH }}
+          key: ${{ env.CACHE_KEY }}-${{ matrix.node-version }}
+      - name: Run Styelint
+        run: yarn run stylelint "**/*.{vue,css,scss}" --ignore-path .gitignore -f verbose
```

Comment:
Could you make this a yarn script in the `package.json` to run instead? 
This will make it easier for us to manage it and keep consistent.